sdk/bootstrap/main.go

```diff
@@ -272,5 +324,9 @@ func main() {
 		log.Printf("Template handling disabled via -template=false: Skipping any config templates")
 	}
 
-	log.Printf("SDK bootstrap successful.")
+	if (args.installCerts) {
+		installCerts()
+	}
```

Comment:
Does it make sense for this to continue on with success  when cert installation was requested but fails?